cmd/skaffold/app/cmd/dev.go

```diff
@@ -105,6 +105,7 @@ func dev(out io.Writer, ui bool) error {
 					}
 				}
 			}
+			r.RPCServerShutdown()
```

Comment:
We should probably return an error and log it if there is an issue. 
Config change also requires a bit more thought: 
- on config change we shutdown the server? How will the IDE know that it needs to reconnect? Can we keep the server alive? I think that would be beneficial. 
- `ConfigurationChanged` probably should be another event 
I added an issue for this #1695.

Reply:
yeah this should be addressed, I think the right solution is to detach the server handling from the runner so that when we create a new one on config change skaffold knows to not do anything with the server. I think I'll address this in a follow up PR